lib/presenters/slack/full_message.rb

```diff
@@ -5,15 +5,20 @@ def execute(applications_by_team:)
         applications = applications_by_team.fetch(:applications)
         "#{url_for_team(applications_by_team)} have #{pull_requests_count(applications)} Dependabot PRs open on the following apps:
 
-#{body(applications).join(' ')}"
+#{body(applications).join('
+')}"
```

Comment:
Maybe a nicer solution to this would be to create the list of applications separately, and then add it to the final message.
Something like this
```suggestion
        body_applications = body(applications).join("\n")
        "#{url_for_team(applications_by_team)} have #{pull_requests_count(applications)} Dependabot PRs open on the following apps:

#{body_applications}"
```

Reply:
Thank you, that's a brilliant suggestion, makes it look so much neater. I have added your change, and amended the commit.